src/components/subscriptions/details/Drawer.js

```diff
@@ -201,10 +220,21 @@ function SubscriptionDrawer(props) {
 }
 
 function SubscriptionHeader(props) {
-    const { username } = props;
+    const { portalId, username } = props;
     const classes = useStyles();
+    const baseURL = constants.DEFAULT_USER_PORTAL_URL;
+    const subURL = navigationConstants.ADMIN_USER_PORTAL_USERS;
+    const linkToUserPortal = `${baseURL}${subURL}/${portalId}`;
     return (
         <div className={classes.drawerHeader}>
+            {portalId && (
+                <Tooltip title={linkToUserPortal}>
+                    <IconButton href={linkToUserPortal} target="_blank">
```

Comment:
When using `target="_blank"`, the `rel="noopener noreferrer"` attribute should also be used.

For example:
https://github.com/cyverse-de/sonora/blob/682a20b94486a97008b5b053f14f2512dd241751/src/components/utils/ExternalLink.js#L10-L12
See also https://v4.mui.com/components/links/#security

Reply:
Thank you for sharing this! I will implement.